packages/streamline/src/components/cards/card-carousel/card-carousel.tsx

```diff
@@ -67,31 +67,37 @@ export const CardCarousel = (props: CardCarouselProps) => {
         style={StyleSheet.absoluteFillObject}
       />
       <Box
-        flexDirection="row-reverse"
+        flexDirection="row"
         justifyContent="space-between"
         alignItems="center"
       >
-        <ButtonIcon
-          testID={`${testID}-close-button`}
-          isLoading={isLoading || isResolving}
-          iconName="Cross"
-          accessibilityLabel="close"
-          appearance="light"
-          onPress={onClose}
-          withContainer
-        />
+        <Box>
+          {isSmall && props.iconName ? (
+            <Icon iconName={props.iconName} color="PURE_WHITE_1000" />
+          ) : null}
 
-        {size === 'large' && props.tag ? (
-          <Tag
-            text={props.tag.text}
-            iconName={props.tag.iconName}
-            appearance="dark"
-          />
-        ) : null}
+          {!isSmall && props.tag ? (
```

Comment:
For consistency, remove usage of `props.xx` and use directly `iconName`, `tag`.

Reply:
The component can take 2 different types of props, `tag` is present only in one of them. I don't know how to do to use only `tag` when it is possibly not present.